pkg/aws/common_validator.go

```diff
@@ -0,0 +1,79 @@
+package aws
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/dylanmei/iso8601"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
This is the label value validation function. It  is generic so we can also use it for other resources if we want to.